docs/iota.go/reference/api_broadcast_transactions.md

```diff
@@ -0,0 +1,38 @@
+# API -> BroadcastTransactions()
+BroadcastTransactions broadcasts a list of attached transaction trytes to the network. Tip-selection and Proof-of-Work must be done first by calling GetTransactionsToApprove and AttachToTangle or an equivalent attach method.  You may use this method to increase odds of effective transaction propagation.  Persist the transaction trytes in local storage before calling this command for first time, to ensure that reattachment is possible, until your bundle has been included.
+> **Important note:** This API is currently in Beta and is subject to change. Use of these APIs in production applications is not supported.
+
+
+## Input
+
+| Parameter       | Type | Required or Optional | Description |
+|:---------------|:--------|:--------| :--------|
+| trytes |  | true | The Trytes to broadcast.  |
```

Comment:
Should this be a []Trytes type as well? In general, the Input part seems to be missing parameter types when it's a list of values. Like URIs in addNeighbors as well. Do we just want to do a [] if it's not typed to anything, like uris?

Reply:
Fixed. There was a problem where variadic parameters were not gathered correctly.